integrations/android_studio.rst

```diff
@@ -0,0 +1,217 @@
+.. _android_studio:
+
+
+|android_studio_logo| Android Studio
+____________________________________
+
+
+Use can use Conan to :ref:`cross-build your libraries for Android<cross_building_android>` in different architectures.
+If you are using Android Studio for your Android application development, you can integrate it conan to automate the
+library building for the different architectures that you want to support in your project.
+
+Here is an example of how to integrate the ``libpng`` conan package library in an Android application, but any library
+that can be cross-compiled to Android could be used using the same procedure.
+
+We are going to use the "Hello World" application as an example:
```

Comment:
Not clear, you are not doing a Hello World, you are doing an app with libpng, zlib.